app/js/main.js

```diff
@@ -128,6 +140,17 @@ function startup() {
 
     // Set default view
     switchMode("capture");
+    
+    //Load top menu
```

Comment:
Comments should have one space after the second slash and before the first character.